exporter/elasticsearchexporter/README.md

```diff
@@ -213,3 +213,89 @@ Other metric types (Histogram, Exponential Histogram, Summary) are ignored.
+
+"Preserved" means the convention should be preserved in ECS mode. This can happen when an attribute needs to be mapped to an ECS equivalent but also preserved in its original form.
+
+| Semantic Convention Name | ECS Name | Preserve |
+|--------------------------|----------|----------|
+| cloud.platform           | cloud.service.name | false |
+| container.image.tags     | container.image.tag | false |
+| deployment.environment   | service.environment | false |
+| host.arch                | host.architecture | false |
+| host.name                | host.hostname | true |
+| k8s.deployment.name      | kubernetes.deployment.name | false |
+| k8s.namespace.name       | kubernetes.namespace | false |
+| k8s.node.name            | kubernetes.node.name | false |
+| k8s.pod.name             | kubernetes.pod.name | false |
+| k8s.pod.uid              | kubernetes.pod.uid | false |
+| os.description           | host.os.full | false |
+| os.name                  | host.os.name | false |
+| os.type                  | host.os.platform | false |
+| os.version               | host.os.version | false |
+| process.executable.path  | process.executable | false |
+| process.runtime.name     | service.runtime.name | false |
+| process.runtime.version  | service.runtime.version | false |
+| service.instance.id      | service.node.name | false |
+| telemetry.distro.name    | "" | false |
+| telemetry.distro.version | "" | false |
+| telemetry.sdk.language   | "" | false |
+| telemetry.sdk.name       | "" | false |
+| telemetry.sdk.version    | "" | false |
+
+### Compound Mapping
+
+There are ECS fields that are not mapped easily 1 to 1 but require more advanced logic.
+
+#### `agent.name`
+
+The agent name takes the form of a compound name consisting of 3 components:
+- `telemetry.sdk.name` or, if not present, defaults to `otlp`,
+- `telemetry.sdk.language`, defaulting to `unknown` in case it is missing,
+- `telemetry.distro.name`, which is allowed to be empty.
+
+These values are all valid:
+
+| `telemetry.sdk.name` | `telemetry.sdk.language` | `telemetry.distro.name` | `agent.name`           |
+|----------------------|--------------------------|-------------------------|------------------------|
+| ""                   | ""                       | ""                      | `otlp`                 |
+| ""                   | dotnet                   | ""                      | `otlp/dotnet`          |
+| opentelemetry        | dotnet                   | ""                      | `opentelemetry/dotnet` |
+| ""                   | java                     | parts-unlimited-java    | `otlp/java/parts-unlimited-java` |
+| ""                   | ""                       | parts-unlimited-java    | `otlp/unknown/parts-unlimited-java` |
+
+#### `agent.version`
+
+Takes the value of `telemetry.distro.version` or `telemetry.sdk.version`.
```

Comment:
Is there a precedence?

Reply:
yes in the order i mentioned them. do you think we could make it more clear/explicit?

Reply:
made it explicit